src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -352,17 +394,21 @@ public String getPetSaEmail() {
     return petSAEmail;
   }
 
-  public UserCredentials getUserCredentials() {
-    return userCredentials;
+  public GoogleCredentials getGoogleCredentials() {
```

Comment:
Maybe make this `Optional<GoogleCredentials>` since the field is nullable? Or is it only null during initialization?

Reply:
Right, but you can wrap it with `Optional.ofNullable()` in the getter for safety.